partials/content-single.php

```diff
@@ -65,17 +65,17 @@
 		echo apply_filters( 'the_content', $post->post_content );
 	}
 	if ( $display_about_the_author && $authors ) {
-		// TODO: add better check to display About the Authors section if at least one author has more than just a name & profile pic?
 		?>
-		<div class="contributors">
+		<hr class="before-contributors clear">
```

Comment:
Add `<hr>` with classes which resembles what we do now for footnotes

Reply:
@SteelWagstaff I like that better since it is consistent. Keep in mind that this may impact on this merged PR #856 however.

Reply:
Maybe add an extra check here? https://github.com/pressbooks/pressbooks-book/blob/283d7d0bd2903dedbd0e2dc6be7bde0b4b1a459f/assets/src/scripts/collapse-sections.js#L65